gtk/docs.md

```diff
@@ -366,3 +366,74 @@ let dialog = FileChooserDialog::with_buttons::<Window>(
     &[("_Cancel", ResponseType::Cancel), ("_Open", ResponseType::Accept)]
 );
 ```
+# DrawingArea
```

Comment:
I think this line should should be switched with the next line iirc.

Reply:
Or removed

Reply:
Even better!